packages/shorebird_cli/lib/src/executables/patch.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:path/path.dart' as p;
+import 'package:scoped/scoped.dart';
+import 'package:shorebird_cli/src/cache.dart';
+import 'package:shorebird_cli/src/shorebird_process.dart';
+
+/// A reference to a [PatchProgram] instance.
+final patchProgramRef = create(PatchProgram.new);
+
+/// The [PatchProgram] instance available in the current zone.
+PatchProgram get patchProgram => read(patchProgramRef);
```

Comment:
I guess just calling it `Patch` would probably cause a naming conflict with our model object. Maybe `PatchExecutable` instead? @felangel wdyt?

Reply:
Renamed to `PatchExecutable`